x-pack/legacy/plugins/xpack_main/server/lib/setup_xpack_main.js

```diff
@@ -16,12 +16,16 @@ import { XPackInfo } from './xpack_info';
  * @param server {Object} The Kibana server object.
  */
 export function setupXPackMain(server) {
-  const info = new XPackInfo(server, {
-    pollFrequencyInMillis: server.config().get('xpack.xpack_main.xpack_api_polling_frequency_millis')
-  });
+  const info = new XPackInfo(server, { licensing: server.newPlatform.setup.plugins.licensing });
 
   server.expose('info', info);
-  server.expose('createXPackInfo', (options) => new XPackInfo(server, options));
+  server.expose('createXPackInfo', (options) => {
```

Comment:
@chrisronline I need your help to test that it doesn't break https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/monitoring/server/init_monitoring_xpack_info.js#L15
There are no tests to verify it

Reply:
@restrry This LGTM. I pulled down the PR and the behavior seems consistent

Reply:
Okay, so this actually wasn't the case. I missed something and this is causing a problem in 7.6